### PR TITLE
mne-python version bump

### PR DESCRIPTION
**Pull request by TheChymera**
and distfile naming correcton

### CODE REVIEW

File: sci-biology/mne-python/mne-python-0.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
```
Comment (jlec) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: 6

File: sci-biology/mne-python/mne-python-0.11.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+S="${WORKDIR}/mne-python-${PV}"
+
+run_test() {
+	PYTHONPATH=. MNE_SKIP_SAMPLE_DATASET_TESTS=1 nosetests -v mne
+}
+
+python_test() {
+	distutils_install_for_testing
+	esetup.py install --root="${T}/test-${EPYTHON}" --no-compile
```
Comment (jlec) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: this isn't needed

File: sci-biology/mne-python/mne-python-0.11.ebuild [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	# Link to test data that won't be included in the final installation
+	local TEST_DIR="${T}/test-${EPYTHON}/$(python_get_sitedir)"
+	cd "${S}" || die
+	find . -type d -name data -exec ln -s "${S}"/{} ${TEST_DIR}/{} \; || die
+	cd ${TEST_DIR} || die
+	VIRTUALX_COMMAND="run_test"
+	virtualmake
```
Comment (jlec) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: This is deprecated. You should use ```virtx() ```

Comment (TheChymera) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: if I replace that line with `virtx()` I get:

```
repoman commit
!!! Section 'neurogentoo' in repos.conf has name different from repository name 'science' set inside repository

RepoMan scours the neighborhood...
/home/chymera/src/neurogentoo/sci-biology/mne-python/mne-python-0.11.ebuild: line 55: syntax error near unexpected token `}'
/home/chymera/src/neurogentoo/sci-biology/mne-python/mne-python-0.11.ebuild: line 55: `}'
 * ERROR: sci-biology/mne-python-0.11::science failed (depend phase):
 *   error sourcing ebuild
 * 
 * Call stack:
 *   ebuild.sh, line 624:  Called die
 * The specific snippet of code:
 *   			source "$EBUILD" || die "error sourcing ebuild"
 * 
 * If you need support, post the output of `emerge --info '=sci-biology/mne-python-0.11::science'`,
 * the complete build log and the output of `emerge -pqv '=sci-biology/mne-python-0.11::science'`.
 * Working directory: '/usr/lib64/python2.7/site-packages'
 * S: '/var/tmp/portage/sci-biology/mne-python-0.11/work/mne-python-0.11'
!!! getFetchMap(): aux_get() error reading sci-biology/mne-python-0.11; aborting.
Unable to generate manifest.
```

Comment (SoapGentoo) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: You call a function in bash with just `virtx`, saying `virtx()` is just used to denote a function

File: sci-biology/mne-python/mne-python-0.11.ebuild [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+python_test() {
+	distutils_install_for_testing
+	# Link to test data that won't be included in the final installation
+	local TEST_DIR="${T}/test-${EPYTHON}/$(python_get_sitedir)"
+	cd "${S}" || die
+	find . -type d -name data -exec ln -s "${S}"/{} ${TEST_DIR}/{} \; || die
+	cd ${TEST_DIR} || die
+	VIRTUALX_COMMAND="run_test"
+	virtx
```
Comment (jlec) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: You should test what you commit, this doesn't do what you think it does.

Comment (TheChymera) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: I actually tested it and it emerges fine. From what I can see in equery it also installed what it should be installing - wht am I missing?

Comment (jlec) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: But no test are run, aren't they?

Comment (TheChymera) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: hmmmm.... ok, so I replace the last two lines with `virtx "run_test"`? qgrep doesn't show me many examples on this,,,

Comment (jlec) [inline sci-biology/mne-python/mne-python-0.11.ebuild]: https://devmanual.gentoo.org/ this is our manual and [here](https://devmanual.gentoo.org/eclass-reference/index.html) you can find the references of our eclasses.

**Pull request closed without merging**